src/09-workorders/work-orders.apib

```diff
@@ -1,5 +1,51 @@
 ## Work Orders [/workOrders]
 
+### workOrders.info [GET /workOrders.info]
+
+Get details for a single work order
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `cde0bc5f-8602-4e12-b5d3-f03436b54c0d` (string, required)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string)
```

Comment:
The `id` returned should be equal to the one passed in the request.

Reply:
Makes sense